composer.json

```diff
@@ -35,7 +37,10 @@
     "autoload": {
         "psr-4": {
             "Raygun4php\\": "src/Raygun4php/",
-            "Raygun4php\\Tests\\": "tests/"
+            "Raygun4php\\Tests\\": "tests/",
+            "Raygun4php\\Tests\\Stubs\\": "tests/stubs",
```

Comment:
Wouldn't this be implied by the previous line?

For the two below it'd be the same thing, if you rename your new folders to `src/Raygun4php/Interfaces` etc, then PSR-4 does its job for us

Reply:
Well spotted!